charts/kubearchive/templates/api_server/certificates.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   renewBefore: 360h  # 15 days
   subject:
     organizations:
-      - {{ .Release.Name }}
+      - {{ required "An organization in the api server certificate must be specified" .Release.Name }}
```

Comment:
`api server -> `API Server` . To maintain consistent.

Reply:
Fixed!